### PR TITLE
remove deepequals in tests

### PR DESCRIPTION
**Pull request by ramaraochavali**
**Please provide a description of this PR:**

### CODE REVIEW

**Comment by ramaraochavali** [comment]
@wulianglongrd  PTAL. Sorry I missed this in review

**Review by hzxuzhonghu — APPROVED** [review]

File: pilot/pkg/networking/core/v1alpha3/envoyfilter/util_test.go [hunk 1]
```diff
@@ -94,28 +93,19 @@ func TestReplaceAndInsert(t *testing.T) {
...
-			if !reflect.DeepEqual(c.insertBefore, got) {
+			if !slices.Equal(c.insertBefore, got) {
 				t.Errorf("insertBeforeFunc: want %+v but got %+v", c.insertBefore, got)
 			}
-			if !reflect.DeepEqual(c.applied, applied) {
-				t.Errorf("insertBeforeFunc: want %+v but got %+v", c.applied, applied)
-			}
-
-			// insertAfterFunc
-			got, applied = insertAfterFunc(slices.Clone(c.input), f)
```
Comment (wulianglongrd) [inline pilot/pkg/networking/core/v1alpha3/envoyfilter/util_test.go:114]: Thanks, the `insertAfterFunc` test was removed. Was it deleted accidentally?

**Comment by wulianglongrd** [comment]
> @wulianglongrd PTAL. Sorry I missed this in review

Thanks for improving this issue.

File: pilot/pkg/networking/core/v1alpha3/envoyfilter/util_test.go (hunk 1 above)
Comment (ramaraochavali) [reply pilot/pkg/networking/core/v1alpha3/envoyfilter/util_test.go:114]: No. I will fix it

**Pull request merged**